tests/khmer_tst_utils.py

```diff
@@ -53,7 +53,27 @@ def cleanup():
     cleanup_list = []
 
 
+def scriptpath(scriptname='interleave-reads.py'):
```

Comment:
On Mon, Jul 27, 2015 at 09:48:23AM -0700, Michael R. Crusoe wrote:
> > @@ -53,7 +53,27 @@ def cleanup():
> >      cleanup_list = []
> >  
> >  
> > +def scriptpath(scriptname='interleave-reads.py'):
> 
> Why the default value for scriptname? Seems dangerous.

See comment immediately below.  We need something, but it doesn't matter what.
Better approaches welcome, but passing in exactly the scriptname we need
is ugly and will probably fall afoul of copy-pasta.